core/quantity/README.md

```diff
@@ -2,10 +2,69 @@
 
 Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md for license terms and full copyright notice.
 
-## Description
-
 The __@itwin/core-quantity__ package contains classes for quantity formatting and parsing.
 
-## Documentation
+## Terms and Concepts
+
+### Common Terms
+
+- [Unit]($quantity)/[UnitProps]($quantity) - A named unit of measure which can be located by its name or label. The definition of any unit is represented through it's `UnitProps`.
+- [UnitsProvider]($quantity) - A class that will also locate the UnitProps for a unit given name or label. This class will also provide a [UnitConversion]($quantity) to convert from one unit to another.
+- Unit Family/[Phenomenon]($ecschema-metadata) - The physical quantity that this unit measures (e.g., length, temperature, pressure).  Only units in the same phenomenon can be converted between.
```

Comment:
what is 'this unit'?  Should this be written as 'A physical quantity that can be measured'

Reply:
that actually was copy pasted from the other existing quantity formatting doc, I'll fix this here and in the other spot

Reply:
Okay, fixed